src/lib/MapCards.svelte

```diff
@@ -16,7 +16,8 @@
     'FIRST_PUBLISHED'
   export let verified: boolean | undefined = undefined
   export let maxCards: number | undefined = undefined // max amount of cards to show
-  export let playlistId: string | undefined = undefined
+  export let playlistId: number | undefined = undefined
```

Comment:
The issue here is that the other locations pass in a string because they pass it in as playlistId="123456", changing those to playlistId={123456} will solve it. I changed the way it is being passed in now in the respective locations since numbers are more restrictive and appropriate for the playlistId.